browser/src/control/Control.JSDialog.js

```diff
@@ -1028,6 +1028,21 @@ L.Control.JSDialog = L.Control.extend({
 		var keyCode = event.keyCode;
 
 		switch (keyCode) {
+		case 13:
+			// ENTER
+			var dialogKeys = Object.keys(this.dialogs);
+			if (dialogKeys.length) {
+				var lastKey = dialogKeys[dialogKeys.length - 1];
+				var dialogInfo = this.dialogs[lastKey];
+				if (dialogInfo && dialogInfo.text === 'Find and Replace') {
+					var defaultButton = dialogInfo.container.querySelector('#' + dialogInfo.defaultButtonId + ' button');
```

Comment:
best to avoid queries, better to store a button node reference in dialogInfo for example, when it's created

Reply:
(I see this code is repeated few times already so it would be reusable then)